src/components/PokeCard/PokeCard.test.jsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react'
+import PokeCard from './PokeCard'
+
+test('Should render pokemon information on card', () => {
+    render(
+        <PokeCard />
+    )
```

Comment:
It appears we are missing an object to match the props within our PokeCard component such as: { url_image, pokemon, id, type_1, type_2 }. Be sure to add in fake data so that we can test to ensure more than just the html is being rendered on the page.